response_operations_ui/views/messages.py

```diff
@@ -137,12 +139,41 @@ def _get_from_name(message):
         msg_from = message['@msg_from']
         return f"{msg_from.get('firstName')} {msg_from.get('lastName')}"
     except KeyError:
-        return message.get('msg_from')
+        logger.exception("Failed to retrieve message from name", message_id=message['msg_id'])
+        return 'Unavailable'
 
 
 def _get_to_name(message):
     try:
-        name = message.get('@msg_to')[0].get('firstName') + ' ' + message.get('@msg_to')[0].get('lastName')
-    except IndexError:
-        name = message.get('msg_to')[0]
-    return name
+        if message.get('msg_to')[0] == 'GROUP':
+            return f"{get_survey_short_name_by_id(message.get('survey'))} Team"
```

Comment:
Do we want to add a log here to know if this is getting called?

Reply:
I don't think so, this is normal behaviour for if a respondent creates a message. If the survey service is called logs are generated then.